allauth/account/tests/test_login.py

```diff
@@ -334,6 +334,26 @@ def test_account_authenticated_login_redirects_is_false(self):
         resp = self.client.get(reverse("account_login"))
         self.assertEqual(resp.status_code, 200)
 
+    @override_settings(
+        ACCOUNT_AUTHENTICATED_LOGIN_REDIRECTS=False,
+        ACCOUNT_EMAIL_VERIFICATION=app_settings.EmailVerificationMethod.OPTIONAL,
+    )
+    def test_login_while_authenticated(self):
```

Comment:
The test suite is moving towards pytest, I would like to put a stop top adding new class based tests. Can this be converted?

Reply:
I think I converted it to a pytest version, but I'm not 100% sure as I've never used pytest-django before. Please let me know if anything has to be fixed.